tik/lib/include/tik/Util.h

```diff
@@ -22,9 +29,10 @@ namespace TraceAtlas::tik
     std::set<llvm::BasicBlock *> GetReachable(llvm::BasicBlock *base, std::set<int64_t> validBlocks);
     bool IsSelfReachable(llvm::BasicBlock *base, const std::set<int64_t> &validBlocks);
     bool IsReachable(llvm::BasicBlock *base, llvm::BasicBlock *target, const std::set<int64_t> &validBlocks);
-    std::set<llvm::BasicBlock *> GetEntrances(std::set<llvm::BasicBlock *> &blocks);
-    std::set<llvm::BasicBlock *> GetExits(std::set<llvm::BasicBlock *> blocks);
-    std::set<llvm::BasicBlock *> GetExits(llvm::Function *F);
+    std::set<llvm::BasicBlock *> GetEntrances(const std::set<llvm::BasicBlock *> &);
+    std::set<llvm::BasicBlock *> GetExits(std::set<llvm::BasicBlock *> &, llvm::BasicBlock *);
+    std::set<llvm::BasicBlock *> GetExits(std::set<llvm::BasicBlock *>);
+    std::set<llvm::BasicBlock *> GetExits(llvm::Function *);
```

Comment:
Same question, why were all the argument names removed?

Reply:
The names and the types are the only bits passed to doxygen. With them removed it makes these functions very anonymous.

Reply:
Same reason as above, but ill change it back